### ISSUE TITLE
"nx connect-to-nx-cloud" doesn't seem like is a command, should it be just "nx connect"?

### ISSUE THREAD

**Issue by bradtaniguchi** [nrwl/nx#13385]
### Documentation issue

- [ ] Reporting a typo
- [x] Reporting a documentation bug
- [ ] Documentation improvement
- [x] Documentation feedback

### Is there a specific documentation page you are reporting?

https://nx.dev/core-features/distribute-task-execution

### Additional context or description

The linked docs page specifies to run the following command to get going with nx cloud:
```bash
nx connect-to-nx-cloud
```

except doing so results in the default app trying to run the `connect-to-nx-cloud` target, as if it was any normal command such as `nx lint` (which would run the project+target `my-project:lint`)

Googling around it seems like the correct command is just:
```bash
nx connect
```

Running this seems to get the correct behavior to connect to the nx cloud.

@nrwl/* packages in package.json at time of error:
```
    "@nrwl/cli": "15.2.1",
    "@nrwl/eslint-plugin-nx": "15.2.1",
    "@nrwl/jest": "15.2.1",
    "@nrwl/js": "15.2.1",
    "@nrwl/linter": "15.2.1",
    "@nrwl/next": "^15.2.1",
    "@nrwl/node": "15.2.1",
    "@nrwl/nx-cloud": "^15.0.2",
    "@nrwl/react": "15.2.1",
    "@nrwl/storybook": "15.2.1",
    "@nrwl/web": "15.2.1",
    "@nrwl/workspace": "15.2.1",
```

I think the command must of changed?

**Comment by FrozenPandaz** [comment]
Yes, can you amend the docs please?

**Comment by bradtaniguchi** [comment]
I can fix this and #13388 at the same time.

👍🏼
